### PR TITLE
Anushka

### PR DESCRIPTION
**Pull request by Anushkaguptaaaa**

### CODE REVIEW

File: lib/custom_widgets/loginbtn.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+class loginbtn extends StatelessWidget {
```
Comment (MUDITJAINN) [inline lib/custom_widgets/loginbtn.dart]: change this to button

File: lib/custom_widgets/guest.dart [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class guest extends StatelessWidget {
```
Comment (MUDITJAINN) [inline lib/custom_widgets/guest.dart]: change name to button2

File: lib/custom_widgets/guest.dart [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      ),
+      onPressed: () {
+        if (callback != null) {
+          callback!();
+        }
+      },
+      child: const SizedBox(
+        width: double.infinity,
+        child: Text(
+          'Continue as guest',
```
Comment (MUDITJAINN) [inline lib/custom_widgets/guest.dart]: pass text as a variable

File: lib/custom_widgets/inputfield_widget.dart [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+class inputfield_widget extends StatelessWidget {
+
+  final String hintText;
+  inputfield_widget(this.hintText);
```
Comment (MUDITJAINN) [inline lib/custom_widgets/inputfield_widget.dart]: add const before constructor

File: lib/custom_widgets/loginbtn.dart [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  final VoidCallback? callback;
+  loginbtn(this.callback);
```
Comment (MUDITJAINN) [inline lib/custom_widgets/loginbtn.dart]: add const

File: lib/custom_widgets/loginbtn.dart [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  @override
+  Widget build(BuildContext context) => ElevatedButton(style: ElevatedButton.styleFrom(
+        backgroundColor: Color.fromRGBO(2, 148, 219, 1)),
+      onPressed: () {
+        callback!();
+      },
+      child: const SizedBox(
+        width: double.infinity,
+        child: Text('Login',
```
Comment (MUDITJAINN) [inline lib/custom_widgets/loginbtn.dart]: add text as a variable

File: lib/custom_widgets/text_widget.dart [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
```
Comment (MUDITJAINN) [inline lib/custom_widgets/text_widget.dart]: remove this widget , use directly Text in login screen

File: lib/screens/login_screen.dart [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+class _Login_screenState extends State<Login_screen> {
+  @override
+  Widget build(BuildContext context)
+  {
+    return Scaffold(
+      body:  Center(
+
+        child: Column(
```
Comment (MUDITJAINN) [inline lib/screens/login_screen.dart:32]: add margin to column instead of container remove margin from inner child container.

**Comment by TheDarkLord100** [comment]
Font of "Welcome to DIT University"
Color of "Email" and "password" text
Border of input field
spacing between input fields
Length of divider
vertical margin of OR line
Signup text in blue color
Continue as guest button border color

File: lib/screens/Signup_screen.dart [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+  final String mail='E-MAIL';
+  final String pass="PASSWORD";
+  final String phoneno="PHONE NUMBER";
+  final String Sapid="SAP ID";
+  final String text1="Already have account? ";
+  final String text2=" Login";
+  @override
+  Widget build(BuildContext context)
+  {
+    return Scaffold(
```
Comment (MUDITJAINN) [inline lib/screens/Signup_screen.dart:25]: blue colour upper design is not present

File: lib/screens/Signup_screen.dart [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+                ),
+              ),
+              const SizedBox(height: 30),
+              inputfield_widget(name,false,CustomColors.backgroundColor),
+
+              const SizedBox(height: 20),
+              inputfield_widget(mail,false,CustomColors.backgroundColor),
+
+              const SizedBox(height: 20),
+              inputfield_widget(phoneno,false,CustomColors.backgroundColor),
```
Comment (MUDITJAINN) [inline lib/screens/Signup_screen.dart]: bg color of textfields

File: lib/screens/Signup_screen.dart [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+              const SizedBox(height: 20),
+              inputfield_widget(pass,true,CustomColors.backgroundColor),
```
Comment (MUDITJAINN) [inline lib/screens/Signup_screen.dart]: change the border colour of textfiles

File: lib/screens/Signup_screen.dart [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+              //input field via password
+              const SizedBox(height: 20),
```
Comment (MUDITJAINN) [inline lib/screens/Signup_screen.dart]: use correct spacing between widgets

File: lib/screens/Signup_screen.dart [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+                  TextButton(onPressed: (){
+                    Navigator.pop(context);
+                  },
+                      child: Text(
+                        text2,
+                        textAlign: TextAlign.center,
+                        style: GoogleFonts.inter(
+                          fontSize:17,
+                          color: CustomColors.blueColor1,
+                          fontWeight:FontWeight.w700,
```
Comment (MUDITJAINN) [inline lib/screens/Signup_screen.dart]: reduce font-weight

File: lib/screens/Signup_screen.dart [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+
+              const SizedBox(height: 20),
+              inputfield_widget(pass,true,CustomColors.backgroundColor),
+
+              //input field via password
+              const SizedBox(height: 20),
+              inputfield_widget(Sapid,false,CustomColors.backgroundColor),
+
+              const SizedBox(height: 40),
+              button(signup,()
```
Comment (MUDITJAINN) [inline lib/screens/Signup_screen.dart]: button shape/radius

**Comment by MUDITJAINN** [comment]
bottom overflow error in the login screen 
login button radius
on-screen rotation overflow signup and login

**Pull request merged**